wgengine/router/router_linux.go

```diff
@@ -175,6 +176,14 @@ func newUserspaceRouterAdvanced(logf logger.Logf, tunname string, linkMon *monit
 		}
 	}
 
+	if !envknob.Bool("TS_DISABLE_MODPROBE") {
```

Comment:
generally we put these envknob calls as package globals for the side effect of them registering their use, which gets logged/reported.